File: matriz5x5.py
FILAS = 5
COLUMNAS = 5
 
 
def llenar_matriz():
    print("Ingrese los 25 valores de la matriz (5x5):\n")
    matriz = []
    for fila in range(FILAS):
        fila_actual = []
        for columna in range(COLUMNAS):
            valor = int(input(f"  Valor [fila {fila + 1}, columna {columna + 1}]: "))
            fila_actual.append(valor)
        matriz.append(fila_actual)
    return matriz
 
 
def mostrar_matriz(matriz):
    print("\n--- Matriz ingresada ---")
    for fila in matriz:
        print("".join(f"{valor:6d}" for valor in fila))
 
 
def main():
    matriz = llenar_matriz()
    mostrar_matriz(matriz)
 
 
if __name__ == "__main__":
    main()

File: matriz5x5.java
import java.util.Scanner;


class Matriz5x5  {
 
    static final int FILAS = 5;
    static final int COLUMNAS = 5;

    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int[][] datos = new int[FILAS][COLUMNAS];
 
        llenarMatriz(datos, teclado);
        mostrarMatriz(datos);
 
        teclado.close();
    }
 
    private static void llenarMatriz(int[][] datos, Scanner teclado) {
        System.out.println("Ingrese los 25 valores de la matriz (5x5):\n");
        for (int fila = 0; fila < FILAS; fila++) {
            for (int col = 0; col < COLUMNAS; col++) {
                System.out.print("  Valor [fila " + (fila + 1) + ", columna " + (col + 1) + "]: ");
                datos[fila][col] = teclado.nextInt();
            }
        }
    }
 
    private static void mostrarMatriz(int[][] datos) {
        System.out.println("\n--- Matriz ingresada ---");
        for (int fila = 0; fila < FILAS; fila++) {
            for (int col = 0; col < COLUMNAS; col++) {
                System.out.printf("%6d", datos[fila][col]);
            }
            System.out.println();
        }
    }
}